v2/manifests/core-ons/zebedee.yml

```diff
@@ -40,7 +40,7 @@ services:
       SERVICE_AUTH_TOKEN:             ${SERVICE_AUTH_TOKEN:?please define a valid SERVICE_AUTH_TOKEN in your local system}
       ENABLE_STATIC_FILES_PUBLISHING: "true"
       FILES_API_URL:                  ${FILES_API_URL:-http://dp-files-api:26900}
-      ENABLE_JWT_SESSIONS:            "false"
+      ENABLE_JWT_SESSIONS:            "true"
```

Comment:
As this is the core manifest, I think we should be setting this as a default that can be overridden.

Reply:
This has now been moved to the cmd stack file